### ISSUE TITLE
Field to save actual MLModel code 

### ISSUE THREAD

**Issue by Taniya-Das** [aiondemand/AIOD-rest-api#248]
#202 
Currently openml mlmodel code is the model name. 
We should find a better way to save the actual ML model code. 
Normally, the actual code should be in the distribution. For some resources (e.g. EducationalResource) we have a .content string. I don't think we should add that to MLModel, it's non-sensical for most MLModels to have a "content" string.

**Comment by PGijsbers** [comment]
So this should be addressed as part of the metadata schema design, correct?

**Comment by Taniya-Das** [comment]
Yes, this needs to be addressed else we will not be able to fetch many of the OpenML pipelines.

**Comment by PGijsbers** [comment]
Could you ask Antonis about it then?
